src/Compilers/CSharp/Test/Emit3/Semantics/PatternMatchingTests_Scope.cs

```diff
@@ -12394,96 +12394,130 @@ void Test14()
 }
 ";
             var compilation = CreateCompilation(source, options: TestOptions.DebugExe, parseOptions: TestOptions.Regular);
-            int[] exclude = new int[] { (int)ErrorCode.ERR_BadVarDecl,
-                                        (int)ErrorCode.ERR_SyntaxError,
-                                        (int)ErrorCode.ERR_UnexpectedToken,
-                                        (int)ErrorCode.WRN_UnreferencedVar,
-                                        (int)ErrorCode.ERR_UseDefViolation
-                                      };
+            int[] exclude = [
```

Comment:
these tests are copies of hte outvar tests, and are equally insane :)

Reply:
Tag yourself, I'm the 'Test11' and 'Test13' cases which are mysteriously commented out